pymc3/stats.py

```diff
@@ -232,6 +237,71 @@ def quantiles(x, qlist=(2.5, 25, 50, 75, 97.5)):
         print("Too few elements for quantile calculation")
 
 
+def df_summary(trace, varnames=None, stat_funcs=None, alpha=0.05, batches=100):
+    """Create a data frame with summary statistics.
+
+    Parameters
+    ----------
+    trace : MultiTrace instance
+    varnames : list
+        Names of variables to include in summary
+    stat_funcs : None or list
+        A list of functions used to calculate statistics. By default,
+        the mean, standard deviation, simulation standard error, and
+        highest posterior density intervals are included.
+
+        The functions will be given one argument, the samples for a
+        variable as a 2 dimensional array, where the first axis
+        correspond to sampling iterations and the second axis
+        represent the flattened variable (e.g., x__0, x__1,...). It
+        should return either
+        1) a Pandas Series instance where each value corresponds to a
```

Comment:
Function passing via a pandas data structure ("pandas" as a name is not capitalized, FWIW) is not entirely clear to me. Maybe add an example?